plugins/HoloColorPicker/build.gradle

```diff
@@ -16,4 +16,5 @@ android {
     lintOptions {
         abortOnError false
     }
+    buildToolsVersion '26.0.2'
```

Comment:
You're still changing the buildToolsVersion here. This is causing the build failure.

Reply:
Oh no, I clicked the update project button once and it seems to have changed the build version everywhere. Do I need to change this and send the PR again?